sources/install/package_wordlists.sh

```diff
@@ -20,11 +20,11 @@ function install_wordlists_apt_tools() {
 
 function install_cewl() {
     colorecho "Installing cewl"
-    rvm use 3.0.0@cewl --create
+    rvm use 3.1.2@cewl --create
```

Comment:
Bad version here

Reply:
crewl is not compatible with the version 3.2.2

Reply:
Let's comment that information, for future reference